### PR TITLE
Add Devtestlabs/2018-09-15

### PR DESCRIPTION
**Pull request by jviau**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

This PR adds devtestlabs/2018-09-15 stable release. This swagger has already been reviewed and approved here: https://github.com/Azure/azure-rest-api-specs-pr/pull/579.

### CODE REVIEW

File: specification/devtestlabs/resource-manager/readme.go.md [hunk 1]
```diff
@@ -35,3 +36,13 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 ``` yaml $(tag) == 'package-2015-05-preview' && $(go)
 output-folder: $(go-sdk-folder)/services/preview/devtestlabs/mgmt/2015-05-21-preview/$(namespace)
 ```
+
+
+### Tag: package-2018-09 and go
```
Comment (jviau) [inline specification/devtestlabs/resource-manager/readme.go.md]: This change is new from the already reviewed and approved PR. I was not sure if I had to add it or not. Let me know if it should be reverted.

File: specification/devtestlabs/resource-manager/readme.ruby.md [hunk 2]
```diff
@@ -24,3 +25,13 @@ Please also specify `--ruby-sdks-folder=<path to the root directory of your azur
 namespace: "Azure::DevTestLabs::Mgmt::V2016_05_15"
 output-folder: $(ruby-sdks-folder)/management/azure_mgmt_devtestlabs/lib
 ```
+
+### Tag: package-2018-09 and ruby
```
Comment (jviau) [inline specification/devtestlabs/resource-manager/readme.ruby.md]: This change is new from the already reviewed and approved PR. I was not sure if I had to add it or not. Let me know if it should be reverted.

**Comment by hovsepm** [comment]
@jviau please rework you PR in a way that first commit is a copy of the existing swagger with the new name and folder structure and the second commit is update with you changes. Otherwise it is not clear what was changed since the last version.

**Review by ryansbenson — APPROVED** [review]
Approved

**Comment by jviau** [comment]
@hovsepm sorry, thought that was only needed for the review PR. I have reworked the commits as requested now.

**Comment by hovsepm** [comment]
>   - tag: package-2016-05

Is there any specific reason Java multi-api does not contain the new API version? 

---
Refers to: specification/devtestlabs/resource-manager/readme.md:146 in 6af1914. [](commit_id = 6af191447abf057ee3216d1f7794014d19256ece, deletion_comment = False)

**Comment by jviau** [comment]
@hovsepm thank you for catching that. I have added it.

**Comment by lmazuel** [comment]
@AutorestCI rebuild azure-sdk-for-java

**Comment by jviau** [comment]
Anything else I need to do before this can be merged in?

**Review by hovsepm — APPROVED** [review]

**Comment by hovsepm** [comment]
For .NEt you will need to create a PR from this swagger to https://github.com/Azure/azure-sdk-for-net repository. Refer to the readme.md file there.

**Pull request merged**